<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>82</y>
      <width>536</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Money</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dSpinMoney"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Months</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinMonths"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Percents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dSpinPercents"/>
     </item>
     <item>
      <widget class="QComboBox" name="depositeSelect">
       <item>
        <property name="text">
         <string>Timed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Comulative</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VIP</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StartBtn">
       <property name="text">
        <string>Start deposite</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>160</y>
      <width>392</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Days from start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinDays"/>
     </item>
     <item>
      <widget class="QPushButton" name="calculateBtn">
       <property name="text">
        <string>Calculate income</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="incomeLabel">
       <property name="text">
        <string>Income</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="takeMoneyBtn">
       <property name="text">
        <string>Take money</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>153</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QDoubleSpinBox" name="dSpinMoneyToAdd"/>
     </item>
     <item>
      <widget class="QPushButton" name="addMoneyBtn">
       <property name="text">
        <string>Add money</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
